docs/v1/api.md

```diff
@@ -1,43 +1,63 @@
 # V1 API
 
-This document contains the Kubernetes resource types for the v1 service catalog.
+This document contains resource types and their standard usage in v1 service
+catalog. Although this API will be implemented in Kubernetes, other systems
+are not precluded from implementing it as well.
 
 # Resource Types
 
 This section lists descriptions of the resources in the service catalog API.
 
-_ __Note:__ All names herein are tentative, and should be considered placeholders
+*__Note:__ All names herein are tentative, and should be considered placeholders
 and used as placeholders for purposes of discussion only. This note will be
-removed when names are finalized._
+removed when names are finalized.*
+
+## `Broker`
+
+This resource is created by an administrator to instruct the service catalog's
+controller event loop to do the following:
+
+1. Make a request against a given CF service broker's catalog endpoint
+   (`GET /v2/catalog`)
+2. Translate the response to a list of `ServiceClass`es
+3. Write each translated `ServiceClass` to stable storage
+
+*TODO: should we call out what happens when a `Broker` resource is deleted?*
 
 ## `ServiceClass`
 
-This resource indicates a general kind of backing service that a consumer
-may request. The 'service kind' concept is purposefully arbitrary. We expect
-each cluster operator team to assign specific meaning to the ones they choose.
+This resource is created by the service catalog's controller event loop after
+it has received a `Broker` resource and successfully called the backing CF
+broker's catalog endpoint. It represents a single (serviceID, planID) pair that
```

Comment:
each pair is a separate resource? I thought a plan was going to be a subresource of a serviceclass.

As a check, this reads to me as
```
(x, y)
(x, w)
(x, z)
```
I thought it would be more like:
```
(x, [y,w,z])
```